src/Servers/Connections.Abstractions/src/Features/ITlsHandshakeFeature.cs

```diff
@@ -15,6 +19,18 @@ public interface ITlsHandshakeFeature
     /// </summary>
     SslProtocols Protocol { get; }
 
+#if NET7_0_OR_GREATER
+    /// <summary>
+    /// Gets the <see cref="TlsCipherSuite"/>.
+    /// </summary>
+    TlsCipherSuite? NegotiatedCipherSuite { get => default; }
+
+    /// <summary>
+    /// Gets the <see cref="SslStream"/>.
+    /// </summary>
+    SslStream? SslStream { get => default; }
```

Comment:
SslStream will only ever work on Kestrel, it should be on a separate feature. It's at least theoretically possible for the other servers to implement NegotiatedCipherSuite from the data they have.

Reply:
Do I call it `ISslStreamFeature` or `ITlsSslStreamFeature`? 🙃